### DISCUSSION TITLE
Hiding certain folders from timeline

### DISCUSSION THREAD

**Discussion by xsirpreisx** [immich-app/immich#4640] [General]
I am completely new here, but I am thinking about getting a new NAS and setting up immich as a replacement for Lightroom (as I have stopped using DSLRs and want to have a web based photo system). 

I have a question: I have all my ~30k images structured in yearly folders, and then subfolders for all of the "events" in that year.

I do, as one example, have a subfolder in the 2017 folder containing all images of our house construction, that I don't really want to see in immichs timeline view. There, I only want "presentable" photos, not details of electrical wires etc.

Is there some kind of way to hide certain folders in the timeline? I tried the immich demo and it looks like the "Archive" could be something that could be used for this, but I'm not sure.

Hope you get what I mean.

**Comment by klaus1k** [comment]
I also vote for this feature.
Now that you can easily upload screenshot albums etc to the server with the new storage template, it would be amazing to have a filter for the timeline.

Or maybe even options per user what to show in the timeline and what not to show. Probably fits well with the partner sharing in timeline issue that is worked on right now.
